File: c/2.1.c
#include <stdio.h>
int main(int argc, char *argv[]){
  int sum = 2;
  int i = 3;
  int a = 1;
  int b = 2;

  while(i < 4000000){
    i = a + b;
    a = b;
    b = i;

    if(i % 2 == 0) {
      sum = sum + i;
    }

  }


  printf("%d\n",sum);

  return 0;
}

File: go/2.1.go
package main

import "fmt"

func main() {
  sum := 2
  a := 1
  b := 2
  for i := 3; i < 4000000; {
    i = a + b
    a = b
    b = i
    if i % 2 == 0 {
      sum += i
    }
  }

  fmt.Println(sum)
}
